src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -8,6 +8,9 @@
 # pylint: disable=line-too-long
 
 helps['ad sp create-for-rbac'] = """
+            parameters:
+                - name: --cert-name
```

Comment:
I definitely don't want anything like "key-vault-cert-name" as it feels too "Xplatty" to me. However, with the refactoring work I've done, I think I will experiment with simply overloading cert to be contextual.